README.md

```diff
@@ -87,7 +87,12 @@ Run the tests
 
 #### Make the release
 
-To be able to publish the package, create a [.pypirc file] containing your pypi username an password.
+To be able to publish the package, install twine and python-dotenv:
+
+    pip install twine
+    pip install python-dotenv[cli]
```

Comment:
Is `python-dotenv[cli]` used for anything else than `dotenv run ./publish-to-pypi.sh`?

If so, we could mention that it is optional to install it, as I assume many of us (everyone?) has [ruby dotenv](https://github.com/bkeepers/dotenv) installed globally (could there be any conflict?)

Reply:
> Is python-dotenv[cli] used for anything else than dotenv run ./publish-to-pypi.sh?

Only that.

> If so not, we could mention that it is optional to install it, as I assume many of us (everyone?) has ruby dotenv installed globally (could there be any conflict?)

It will definitely conflict (it did so locally), for me it liked the Python version more. I tried to pretend I was a Pythonista, why I didn't use Ruby's `dotenv`.

But yes, adding a optional note ought to take care of it.

Reply:
879ee05

Reply:
@dentarg ^ OK?

Reply:
👍